subjects/PE/x86/IncorrectUserSignature/IncorrectUserSignature.c

```diff
@@ -7,6 +7,9 @@
 // 00401120: void test6(Stack (ptr Eq_2) c, Stack int32 a, Stack int32 b)
 void test6(cdecl_class * c, int32 a, int32 b)
 {
+// Incorrect user-defined signature
+"void sum(int, int)" is set instead of "int sum(int, int)"
+Reko should emit code which gives chance for user to find this mistake
```

Comment:
Multiline comments still need some work. Perhaps a separate PR?

Reply:
Yes, it should be done in other PR. This PR could be merged